modules/fundamental/src/advisory/model/details/advisory_vulnerability.rs

```diff
@@ -52,7 +52,7 @@ impl AdvisoryVulnerabilityHead {
 
         if let Some(advisory_vuln) = advisory_vuln {
             Ok(AdvisoryVulnerabilityHead {
-                head: VulnerabilityHead::from_vulnerability_entity(vulnerability),
+                head: VulnerabilityHead::from_vulnerability_entity(vulnerability, tx).await?,
```

Comment:
a-ha!

Reply:
Yes! Now I need it, and I knew the pattern :)